attendance.py

```diff
@@ -0,0 +1,18 @@
+from student import Student
+
+class Attendance:
+    def __init__(self):
+        self.students = [
+            Student('1', 'John Doe'),
+            Student('2', 'Jane Doe'),
+            Student('3', 'Jim Doe')
```

Comment:
Consider fetching the students' data from a database instead of hardcoding it. This would make the application more scalable and maintainable.